### DISCUSSION TITLE
modulenotfounderror

### DISCUSSION THREAD

**Discussion by chalicha** [hlky/stable-diffusion#277] [General]
hello

i am getting this error 

Relauncher: Launching...
Traceback (most recent call last):
  File "scripts/webui.py", line 3, in <module>
    from frontend.frontend import draw_gradio_ui
ModuleNotFoundError: No module named 'frontend'
Relauncher: Process is ending. Relaunching in 1s...

please someone help me

**Comment by huldumadurin** [comment]
I have the same problem when using 'docker compose up' on Windows 10.

`sd | entrypoint.sh: Launching...
sd  | Relaunch count: 77
sd  | Traceback (most recent call last):
sd  |   File "scripts/webui.py", line 3, in <module>
sd  |     from frontend.frontend import draw_gradio_ui
sd  | ModuleNotFoundError: No module named 'frontend.frontend'
sd  | entrypoint.sh: Process is ending. Relaunching in 0.5s...`

You might want to try the solutions suggested in https://github.com/hlky/stable-diffusion/issues/129 but they didn't work for me.
